vendor/ocaml-4.13-extended/parser.mly

```diff
@@ -55,6 +55,7 @@ let mkmod ~loc ?attrs d = Mod.mk ~loc:(make_loc loc) ?attrs d
 let mkstr ~loc d = Str.mk ~loc:(make_loc loc) d
 let mkclass ~loc ?attrs d = Cl.mk ~loc:(make_loc loc) ?attrs d
 let mkcty ~loc ?attrs d = Cty.mk ~loc:(make_loc loc) ?attrs d
+let mkcst ~loc c = Const.mk ~loc:(make_loc loc) c
```

Comment:
Nit, but aren't the other functions named `mk<foo>` where the corresponding constructors are `P<foo>`, indicating that `mkconst` would be a more symmetric name than `mkcst`?

Reply:
Done!